infra/gcp/bash/ensure-staging-storage.sh

```diff
@@ -289,6 +289,25 @@ function ensure_staging_gcb() {
     ensure_project_role_binding "${project}" "${principal}" "roles/cloudbuild.builds.builder"
     ensure_gcs_role_binding "${bucket}" "${principal}" "objectCreator"
     ensure_gcs_role_binding "${bucket}" "${principal}" "objectViewer"
+
+    local sa_name="gcb-image-builder"
+    local sa_email="${sa_name}@${project}.iam.gserviceaccount.com"
+    local sa_principal="serviceAccount:${sa_email}"
+    color 6 "Ensuring ${sa_email} exists and can sign artifacts in project: ${project}"
+    ensure_service_account \
+      "${project}" \
+      "${sa_name}" \
+      "used by prow to build container images and sign artifacts for ${project}"
+
+    ensure_serviceaccount_role_binding \
+        "${sa_email}" \
+        "${sa_principal}" \
+        "roles/iam.serviceAccountTokenCreator"
+
+    ensure_serviceaccount_role_binding \
+        "${sa_email}" \
```

Comment:
```suggestion
    ensure_project_role_binding \
        "${project}" \
```

I thought the error I shared before was related to the service account format but the issue comes from an incorrect function.
Tested:
```
./ensure-staging-storage.sh k8s-staging-infra-tools
Ensuring staging projects... 
Configuring staging project: k8s-staging-infra-tools 
  Ensuring project exists: k8s-staging-infra-tools 
  Ensuring k8s-infra-staging-infra-tools@kubernetes.io are project viewers 
  Ensuring necessary enabled services staging project: k8s-staging-infra-tools 
  Ensuring disabled services for staging project: k8s-staging-infra-tools 
  Ensuring containeranalysis service agent binding removed for staging project: k8s-staging-infra-tools 
  Ensuring serviceAccount:k8s-infra-gcr-vuln-scanning@k8s-artifacts-prod.iam.gserviceaccount.com can view vulnernability scanning results for project: k8s-staging-infra-tools 
  Ensuring staging GCR repo: gcr.io/k8s-staging-infra-tools 
    Ensuring a GCR repo exists for project: k8s-staging-infra-tools 
    Ensuring k8s-infra-staging-infra-tools@kubernetes.io can write to GCR for project: k8s-staging-infra-tools 
    Ensuring GCR admins can admin GCR for project: k8s-staging-infra-tools 
    Ensuring GCS access logs enabled for GCR bucket in project: k8s-staging-infra-tools 
    Enabling logging on gs://artifacts.k8s-staging-infra-tools.appspot.com/...
  Ensuring staging GCS bucket: gs://k8s-staging-infra-tools 
    Ensuring gs://k8s-staging-infra-tools exists and is world readable in project: k8s-staging-infra-tools 
    Ensuring gs://k8s-staging-infra-tools has auto-deletion of 60 days 
    Ensuring GCS admins can admin gs://k8s-staging-infra-tools in project: k8s-staging-infra-tools 
    Ensuring k8s-infra-staging-infra-tools@kubernetes.io can write to gs://k8s-staging-infra-tools in project: k8s-staging-infra-tools 
    Ensuring GCS access logs enabled for gs://k8s-staging-infra-tools in project: k8s-staging-infra-tools 
    Enabling logging on gs://k8s-staging-infra-tools/...
  Ensuring staging GCB 
    Ensuring staging bucket: gs://k8s-staging-infra-tools-gcb 
      Ensuring gs://k8s-staging-infra-tools-gcb exists and is world readable in project: k8s-staging-infra-tools 
      Ensuring gs://k8s-staging-infra-tools-gcb has auto-deletion of 60 days 
      Ensuring GCS admins can admin gs://k8s-staging-infra-tools-gcb in project: k8s-staging-infra-tools 
      Ensuring k8s-infra-staging-infra-tools@kubernetes.io can write to gs://k8s-staging-infra-tools-gcb in project: k8s-staging-infra-tools 
    Ensuring k8s-infra-staging-infra-tools@kubernetes.io can use GCB in project: k8s-staging-infra-tools 
    Ensuring gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com can use GCB in project: k8s-staging-infra-tools 
    Ensuring gcb-image-builder@k8s-staging-infra-tools.iam.gserviceaccount.com exists and can sign artifacts in project: k8s-staging-infra-tools 
    Updated IAM policy for project [k8s-staging-infra-tools].
    @@ -10,6 +10,8 @@
       role: roles/cloudbuild.builds.builder
     - member: serviceAccount:gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com
       role: roles/cloudbuild.builds.builder
    +- member: serviceAccount:gcb-image-builder@k8s-staging-infra-tools.iam.gserviceaccount.com
    +  role: roles/cloudbuild.builds.editor
     - member: serviceAccount:k8s-infra-gcr-vuln-scanning@k8s-artifacts-prod.iam.gserviceaccount.com
       role: roles/containeranalysis.occurrences.viewer
     - member: serviceAccount:service-1017132094926@containerregistry.iam.gserviceaccount.com
Configuring special cases for Release Managers 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-artifact-promoter 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-build-image 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-ci-images 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-cip-test 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-experimental 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-kubernetes 
  Empowering kubernetes-release-test GCB service account to admin GCR 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-releng 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-releng-test 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-publishing-bot 
Done 
```

Reply:
Alright, changed as suggested :+1: